lib/commands/general.js

```diff
@@ -188,6 +188,19 @@ commands.getStrings = async function (language, stringFile = null) {
   return strings;
 };
 
+commands.setUrl = async function (url) {
+  log.debug(`Attempting to set url '${url}'`);
+  if (!this.isWebContext()) {
+    // use xcrun to open deeplink
+    await exec('xcrun', ['simctl', 'openurl', this.opts.udid || this.sim.udid, url]);
```

Comment:
There is a method on `node-simctl` for this command: https://github.com/appium/node-simctl/blob/master/lib/simctl.js#L67